src/main/java/by/custom/utilcalculator/domain/tree/Node.java

```diff
@@ -10,12 +11,17 @@ public class Node {
     private Node parent;
     private final List<Node> children;
     private final Command key;
+    private final Step nextStep;
 
```

Comment:
You can store step number. Step number is a level of the node in tree.
```suggestion
private Int number;
```

Reply:
Let's discuss, how we can use int stepNumber